app/controllers/v0/profile/address_validation_controller.rb

```diff
@@ -29,6 +36,7 @@ def address_params
           :address_pou,
           :address_type,
           :city,
+          :country_name,
```

Comment:
Is country_name a valid param for both `VAProfile::Models::V3::ValidationAddress` and `VAProfile::Models::ValidationAddress` ?

Reply:
No but it's not an a required parameter.